src/editor/src/routes.js

```diff
@@ -0,0 +1,49 @@
+// Returns the URL's paths
+function splitURL(urlPath) {
+    return urlPath.substr(6).split('/');
+}
+
+// Determines whether path is a file
+function isFile(pathname) {
+    return pathname.split('/').pop().indexOf('.') > -1;
+}
+
+// Determines whether path is a directory
+function isDir(pathname) { return !isFile(pathname); }
+
+const editRegex = new RegExp('\/edit(\/.*)/');
+
+export default {
+    init: (workbox) => {
+        // @ts-ignore
+        workbox.routing.registerRoute(
+            editRegex,
+            async ({ url, event }) => {
+                const paths = splitURL(url.pathname);
+
+                let folder = file = redirectURL = "";
+
+                // If URL contains a file
+                if (isFile(paths[paths.length - 1])) {
```

Comment:
Change all this to use something like I showed you with `fs.stat(path, function(err, stats) {...}` and you can use `stats.isFile()` and `stats.isDirectory()`.

Reply:
If you run `npm run build-webserver` it will overwrite the broken docs `index.html` and then you can use my old stuff for testing.